### ISSUE TITLE
Difference in files between install

### ISSUE THREAD

**Issue by jamescowie** [magento/magento2#1356]
Hello,

So I used the recommended way of installing magento via composer: 

`composer create-project --stability=beta --no-install magento/project-community-edition` 

However for Beta 12 I am seeing a different in files based on the way I install. So via magento/magento2 github repository I see: https://github.com/magento/magento2/blob/0.74.0-beta12/app/code/Magento/Backend/Model/Auth.php#L148

Line: 148

```
self::throwException(__('Please correct the user name or password.'));
```

However in my version installed via composer is see: 

```
self::throwException(__('You did not sign in correctly or your account is temporarily disabled.'));
```

Stepping through the composer method I can see that it pulls the data from http://packages.magento.com/ as each package is then broken down into its modules. Should I take the composer ( packages.magento.com ) version as being the golden build ? If so why is there a difference in the files between what is tagged as a specific version ?

**Comment by alankent** [comment]
Right now, packages.magento.com is the "golden build".  GitHub might have things that have not been "officially released" yet. But the idea of packages.magento.com is so people who are interested can play doing things closer to what it will be at GA and provide us feedback. But its a bit loose still because we are not doing full semantic version checking etc for the code released there.

But if you see problems (like different code that you reported) we are interested in understanding why. If its just GitHub is newer, that is fine. If there is something broken then we need to fix it.

As a general guideline, when we hit GA guidance will be "do a git clone of the main repo if you want to create a pull request; use composer to download packages for all other situations". Since people are submitting bug fixes, most people do a git clone today (which is helpful to us). But if you want to mimic production, you should be doing composer based builds and telling us of problems so we can fix them.

Oh, and also note at GA Magento Connect will host the Composer package repo, so you can get paid extensions and CE etc all from the one place. Patches will also be released that way as well. packages.magento.com is to allow people to experiment before Magento Connect is ready.

**Comment by jamescowie** [comment]
Perfect. Just the answer we needed :) in this instance no code breaks just a gift diff that highlighted the difference.

**Comment by alankent** [comment]
If git is tagged but is different to the code in packages.magento.com with the same version number, that is a bug. So this issue should remain open until it is confirmed we are tagging and releasing the exact same version of the code. You have reported you believe the tagged code and the code in packages.magento.com should be the same in this case (but is different).

**Comment by buskamuza** [comment]
@jamescowie , thanks for the report!
We had some technical issues during publication of `0.74.0-beta12` and so, the packages were published with some additional commits comparing to the GitHub version.
In general, publication happens basing on the same code for both packages and GitHub version, in one time, and this should not happen in future.

**Comment by vpelipenko** [comment]
@jamescowie, this issue is fixed during 0.74.0-beta12 publication. You can verify it on the latest code.

**Comment by vnahalpara** [comment]
Finally i got the solution of login issue

issue with magento setup when u install magento first time and add password and confirmation password it's store wrong password to database(i don't know what is an issue at that time.).

Here is solution

go to this file path and \vendor\zendframework\zend-crypt\src\Utils.php

and echo $expected and $actual variable and check both are same or not

if both variable are not match then change line no 35

`if (function_exists('hash_equals')) {
        return hash_equals($expected, $actual);
    }`

To

`if (function_exists('hash_equals')) {
        return true;
    }`

for direct access to magento admin with any password and then change password and revert back code changes

This solution work for me
